ppr-ui/src/composables/mhrInformation/useTransportPermits.ts

```diff
@@ -109,14 +109,14 @@ export const useTransportPermits = () => {
     setMhrTransportPermit({ key: 'newLocation', value: cloneDeep(locationInfo) })
   }
 
-  const isTransportPermitDisabledQS = computed((): boolean =>
-    // QS role check
-    isRoleQualifiedSupplier.value &&
+  const isTransportPermitDisabled = computed((): boolean =>
+    // QS or SBC role check
+    (isRoleQualifiedSupplier.value || isRoleStaffSbc.value) &&
     // PPR Liens check
-    [APIRegistrationTypes.LAND_TAX_LIEN,
+    ([APIRegistrationTypes.LAND_TAX_LIEN,
     APIRegistrationTypes.MAINTENANCE_LIEN,
     APIRegistrationTypes.MANUFACTURED_HOME_NOTICE]
-      .includes(getLienRegistrationType.value as APIRegistrationTypes) &&
+      .includes(getLienRegistrationType.value as APIRegistrationTypes) ||
```

Comment:
Liz confirmed that either condition (Lien check) or Unit Note would block the Transport Permit.